storage/grpc_client.go

```diff
@@ -955,37 +958,44 @@ func (c *grpcStorageClient) RewriteObject(ctx context.Context, req *rewriteObjec
 	return r, nil
 }
 
-// bytesCodec is a grpc codec which permits receiving messages as either
-// protobuf messages, or as raw []bytes.
-type bytesCodec struct {
-	encoding.Codec
+type bytesCodecV2 struct {
+	encoding.CodecV2
```

Comment:
Why embed here?  You'll get nil panics if you forget to implement something.

If you're trying to "show" that you're implementing an interface, you can do:

```go
var _ encoding.CodecV2 = bytesCodecV2{}
```

Reply:
I think this was something that @neild had in the original codec, but I can remove.